src/world/partners.h

```diff
@@ -3,31 +3,41 @@
 
 #include "common.h"
 
+typedef void (*PartnerFunc)(Npc* partner);
+typedef s32 (*PartnerBoolFunc)(Npc* partner);
+typedef s32 (*PartnerUnkFunc)(Npc* partner, s32 arg1);
```

Comment:
Not certain I love these typedefs, honestly

Reply:
discussion in discord